mem0/graphs/configs.py

```diff
@@ -30,6 +30,14 @@ class GraphStoreConfig(BaseModel):
         description="Configuration for the specific data store",
         default=None
     )
+    llm: Optional[LlmConfig] = Field(
+        description="LLM configuration for querying the graph store",
+        default=None
```

Comment:
You should set a default model here if you are not doing it at any other place.

Reply:
Right, good point!!
https://github.com/mem0ai/mem0/blob/752ddc746391f936dad96cb4d61e300e06f95af8/mem0/llms/openai_structured.py#L14

Will this take care of the case that you mentioned?

Reply:
Not exactly because here you would be passing the default config something like this: https://github.com/mem0ai/mem0/blob/main/mem0/configs/base.py#L40.

The line you are mentioning would only set the default model for Openai provider.